images/linux/toolsets/toolset-1804.json

```diff
@@ -224,6 +224,10 @@
         {
             "package": "aws-sam-cli",
             "cmd": "sam"
+        },
```

Comment:
What about Ubuntu 16.04? Can we use the same approach for it?
If not - let's put some comment to the `ansible.sh` that this script is used only on Ubuntu 16.04 to avoid confusing

Reply:
While I would prefer using the same method for both I think it may not be possible due to the ancient versions of python from Ubuntu 16.04 (all EOL!). pipx requires 3.6+ and even if there are probably some older version I doubt they will work fine. 

Shotly, I support this dual approach.